packages/block-editor/src/components/list-view/block.js

```diff
@@ -94,12 +94,14 @@ function ListViewBlock( {
 		blockAriaLabel = isLocked
 			? sprintf(
 					// translators: %s: The title of the block. This string indicates a link to select the locked block.
-					__( '%s link (locked)' ),
+					__( '%1$s (%2$s) link (locked)' ),
```

Comment:
What happened here? The comment does not seem to reflect the string anymore.

Reply:
Fixed.